dcp_stats/project_stats.py

```diff
@@ -0,0 +1,60 @@
+## Prod Project Rereival
+
+import requests, pprint, hca, csv, os
+
+azul_project_link = 'https://service.explore.data.humancellatlas.org/repository/projects?size=200'
+dss_client = hca.dss.DSSClient(swagger_url='https://dss.data.humancellatlas.org/v1/swagger.json')
+
+
+def get_bundles(project_uuid: str):
+    query = { "query": { "bool": { "must": [ { "match": { "files.project_json.provenance.document_id": project_uuid } } ] } } }
+    resp = dss_client.post_search(replica='aws', es_query=query)
+    assert resp.get('total_hits') != 0
+    bundles = [x.get('bundle_fqid') for x in resp.get('results')]
+    return bundles
+
+def get_project_json_uuid(bundle_uuid: str):
+    manifest = dss_client.get_bundle(uuid=bundle_uuid,replica='aws')
+    files = manifest['bundle']['files']
+    found = None
+    for file in files:
+        if 'project' in file.get('name'):
+            found = file
+    return found
+
+def get_date_uploaded(file_uuid: str):
+    file_metadata = dss_client.get_file(uuid=file_uuid,replica='aws')
+    return file_metadata['provenance']['submission_date']
+
+def output_csv(project_list:list):
+    file_path = os.path.dirname(__file__)
+    with open(f'{file_path}/project_stats.csv', 'w') as f:
+        w = csv.DictWriter(f, project_list[1].keys())
+        w.writeheader()
+        for project in project_list:
+            w.writerow(project)
+
+res = requests.get(azul_project_link)
+project_info = []
+
+for x in res.json().get('hits'):
+    project_uuid = x.get('entryId')
+    cellSuspension = [suspension.get('totalCells') for suspension in x.get('cellSuspensions')]
+    cell_count = sum(cellSuspension)
```

Comment:
@hannes-ucsc  @DailyDreaming 
Some Cell Counts are marked as "Unspecified", how should these be handled?
see https://data.humancellatlas.org/explore/projects on the far right column.

Reply:
@amarjandu We might want to look into counting these by opening the files directly and hard-coding the values in `cell_counts.json`, since it's only 7 projects.

@hannes-ucsc @danielsotirhos What do you think?

Reply:
cell_counts.json is a build product and shouldn't be modified manual since that would strip us of our ability to remove and regenerate it. Can we modify the spreadsheets?

Reply:
@hannes-ucsc We could, but we don't have spreadsheets for the projects in `prod` I believe.  It might be good just to populate the `cell_counts.json` in `.attic` then since it's not currently changed by anything else and this would just be a one-time, static update.

Reply:
if required we can change cell_counts manually,  i just wanted to note that there are projects with cell_counts of 0. 

https://docs.google.com/spreadsheets/d/1AKVIPG6TQ0FmQBGg_C41KYQmdcO2IuV9_fslTTzoZ_8/edit#gid=74070310

There are separate sheets for `dev` and `prod`. 
Currently im not using .attic for the `dev` projects, ive been getting the data from Azul/DSS APIs